AppDashboard/lib/app_dashboard_data.py

```diff
@@ -555,14 +628,22 @@ def update_users(self):
           is_user_cloud_admin = self.helper.is_user_cloud_admin(email)
           can_upload_apps = self.helper.can_upload_apps(email)
           owned_apps = self.helper.get_owned_apps(email)
+          dash_layout_settings = self.get_dash_layout_settings(email)
+
+          dash_change = (dash_layout_settings.get("nav") !=
+                         user_info.dash_layout_settings.get("nav")) and \
```

Comment:
It seems strange to me that `dash_change` would only be true if _both_ have changed.